SocialFilter/sections/specific.txt

```diff
@@ -62,7 +62,8 @@ note.calbee.jp,note.linemusic.jp,diehardtales.com,note.com,note.kagg.jp,note.jp,
 !
-veryweb.jp##.c-articleShare
+mart-magazine.com,veryweb.jp##.c-articleShare
+mart-magazine.com##.c-article__snsShare
```

Comment:
This rule is ok instead of the rules below.
```
mart-magazine.com##div[class^="c-article"][class$="Share"]
```

Reply:
ok